lib/ansible/module_utils/linode.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+def get_user_agent(module):
+    """Retrieve a user-agent to send with LinodeClient requests."""
+    try:
+        from ansible.release import __version__ as ansible_version
+    except ImportError:
+        ansible_version = 'unknown'
+    return 'Ansible-{}/{}'.format(module, ansible_version)
```

Comment:
This looks right to me.  I ran a `git grep -i user_agent` and found a few different conventions, but this seems to be the most popular in Ansible.

```
lib/ansible/module_utils/ec2.py:        config.user_agent_extra = 'Ansible/{0}'.format(__version__)
lib/ansible/plugins/inventory/azure_rm.py:        self.add_user_agent('ansible-dynamic-inventory/{0}'.format(release.__version__))
```
...
```
Ansible-gcdns/%s
Ansible-gce_inventory_plugin/%s
ansible-healthcheck/%s
ansible-target_proxy/%s
```
etc.. Sometimes the module provider name is included, sometimes it is not.  Sometimes it just focuses on the feature of Ansible being provided.

Reply:
:+1: :+1: :+1: